src/main/resources/bug_report.md

```diff
@@ -0,0 +1,41 @@
+## Bug description
+Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
+
+For more non-trivial issues, we would also appreciate if you included the following details:
+
+### Running environment
+ - Git Machete plugin version - %macheteVersion%
+ - IDE %ide%
+ - Operating system [e.g. Windows]
+
+### IDEA - event logs
+%logs%
+
+### IDEA - additional info
+%additionalInfo%
+
+### Debug logs
```

Comment:
I'd rather get rid of that section, that's quite unlikely that users will dig for that kind of detail (nor is it likely to help us in debug a lot once a stack trace is attached)